src/java.desktop/share/classes/javax/swing/text/html/CSS.java

```diff
@@ -2604,9 +2672,27 @@ Object toStyleConstants(StyleConstants key, View v) {
             return Float.valueOf(getValue(false));
         }
 
+        @Override
+        public int hashCode() {
+            return Float.hashCode(span);
```

Comment:
Should `hashCode` take into account `percentage` and `lu` fields which are used in `equals`?